### PR TITLE
test: Add test for selection options when pruning

### PR DESCRIPTION
**Pull request by jelly**
An superuser has the option to delete his or users images, test both
options in a new test so we have images to prune.

### CODE REVIEW

File: test/check-application [hunk 1]
```diff
@@ -1628,6 +1628,42 @@ class TestApplication(testlib.MachineCase):
...
+        # Pruning again, should delete all system images
+        b.click("#image-actions-dropdown")
+        b.click("button:contains(Prune unused images)")
+        b.wait_js_func("ph_count_check", ".pf-c-modal-box__body .pf-c-list li", 3)
+        b.click(".pf-c-modal-box button:contains(Prune)")
+        b.wait_js_func("ph_count_check", "#containers-images table tbody", 0)
+
+        # Prune button should now be disabled
+        b.click("#image-actions-dropdown")
+        b.wait_visible("button:contains(Prune unused images).pf-m-disabled")
```
Comment (KKoukiou) [inline test/check-application:1665]: Try to avoid using 'contains' as it won't work in the browser inspector.

**Review by KKoukiou — APPROVED** [review]

File: test/check-application (hunk 1 above)
Comment (marusak) [reply test/check-application:1665]: > as it won't work in the browser inspector.

That is correct, but I still find it nicer to user `:contains()` as it makes tests that much more readable.

Comment (KKoukiou) [reply test/check-application:1665]: I guess it's more of a taste thing then. :whatever

Comment (jelly) [reply test/check-application:1665]: @KKoukiou do you prefer using an hardcoded id? I guess that makes things more predictable?

Comment (KKoukiou) [reply test/check-application:1665]: This would work nicely when copying pasting css selectors to the browser inspector, but really it's not a big deal.

**Pull request merged**